cluster/juju/layers/kubernetes-e2e/actions/test

```diff
@@ -16,7 +17,6 @@ fi
 # get the host from the config file
 SERVER=$(cat /home/ubuntu/.kube/config | grep server | sed 's/    server://')
 
-# Execute the E2E test
-e2e.test -kubeconfig /home/ubuntu/.kube/config -host $SERVER -ginkgo.focus $FOCUS -ginkgo.skip $SKIP 2>&1 | tee /home/ubuntu/${JUJU_ACTION_UUID}.log
+ginkgo -nodes=$PARALLELISM $(which e2e.test) -- -kubeconfig /home/ubuntu/.kube/config -host $SERVER -ginkgo.focus $FOCUS -ginkgo.skip "$SKIP" 2>&1 | tee /home/ubuntu/${JUJU_ACTION_UUID}.log
```

Comment:
It sounds like you were not able to get parallel runs without this extra ginkgo library. Is this an output of any kubernetes builds perhaps that we could also include in a resource for this charm?